src/constants/assetsConstants.js

```diff
@@ -169,3 +169,6 @@ export const ETHERSPOT_DEFAULT_STABLE_LIST = 'EtherspotDefaultStablesList';
 export const FETCHING_DEFAULT_LIST = 'FETCHING_DEFAULT_LIST';
 export const SET_DEFAULT_LIST = 'SET_DEFAULT_LIST';
 export const SET_STABLE_DEFAULT_LIST = 'SET_STABLE_DEFAULT_LIST';
+
+export const PILLAR_DEFAULT_LIST = 'Pillar Default List';
```

Comment:
We don't use constants for titles, this should be added to the localization then pulled from `translations/translate`